source/gui/settingsDialogs.py

```diff
@@ -1623,6 +1623,30 @@ def makeSettings(self, settingsSizer):
 		wordWrapText = _("Avoid splitting &words when possible")
 		self.wordWrapCheckBox = sHelper.addItem(wx.CheckBox(self, label=wordWrapText))
 		self.wordWrapCheckBox.Value = config.conf["braille"]["wordWrap"]
+		# Translators: The label for a setting in braille settings to select how the context for the focus object should be presented on a braille display.
+		focusContextPresentationLabelText = _("Focus context presentation:")
+		self.focusContextPresentations=[
+			("fill",
+			# Translators: The label for a braille focus context presentation setting that
+			# shows as much as possible focus context information if the focus object doesn't fill up the whole display.
+			# This was the pre NVDA 2017.3 default.
+			_("Always fill display")),

```

Comment:
nit: While I understand you're trying to get the translators comment as close to the translatable string as possible, this does make it a bit hard to read this list. Happy to discuss, but i think this looks more readable despite separating it a bit:

> ```
> # Translators: Blah blah blah
> ("fill", _("Always fill display")),
> ```

I'm pretty sure gettext will still match this - I'm pretty sure we do it elsewhere - but maybe I should check...